production/helm/promtail/values.yaml

```diff
@@ -26,6 +26,9 @@ nameOverride: promtail
 ## ref: https://kubernetes.io/docs/user-guide/node-selection/
 nodeSelector: {}
 
+pipelineStages:
+- docker: {}
```

Comment:
Why docker was the default? Seems to me a bad idea cause it will try to parse json for all entry and most of them aren’t !

I'm thinking that default should be an empty list.

I do like the template change though, you share the same pipeline and that's great

Reply:
now I know why it's docker got a conversation with @slim-bean . I'm wondering if this will be the same requirements for everyone.

Reply:
I agree with you @Kuqd. The default being docker does cause issues for people not running docker like #260. Hence why I submitted PR for [autodetection](https://github.com/grafana/loki/pull/380) of log format that we can use as the new default. Need to first rework it for the new pipeline stuff, just waiting for the dust to settle first.